web-admin/tests/setup/setup.ts

```diff
@@ -183,7 +183,7 @@ setup.describe("global setup", () => {
   setup("should deploy the OpenRTB project", async ({ adminPage }) => {
     // Deploy the OpenRTB project
     const { match } = await spawnAndMatch(
-      "rill",
+      "../rill",
```

Comment:
I think this will be prone to mistake/regression. Rather, can we alias the test's binary as `rill` at the start of the tests?